### DISCUSSION TITLE
Can Announcement Bars Be Page Specific?

### DISCUSSION THREAD

**Discussion by mrworthington** [quarto-dev/quarto-cli#10952] [Q&A] [answered]
### Description

This isn't a bug or something that needs a reprex. Right now, it seems that if you want to use the announcement bar, you are limited to rendering it across the entire site. I have a use case where I'm wanting to limit the announcement bar to two pages on my site (which has way more than two pages) and, currently, I'm struggling to clarify the answer to my question on this repo's previously existing issues, discussion, and via general searching on Google and Stack Overflow.

**Comment by mcanouil** [comment]
See

- https://github.com/quarto-dev/quarto-cli/discussions/10320#discussioncomment-10082311

**Reply by mrworthington** [reply]
Thanks for the quick workaround!

**Comment by mrworthington** [comment, marked as answer]
I tried @mcanouil's solution, but realized it was much easier to add this chunk at the bottom of pages I wanted to hide the announcement alert. This worked really well:

````md
```{ojs}
html`<style>

  #quarto-announcement {
    display: none !important;
  }
  </style>
```
````

**Reply by mcanouil** [reply]
Why are you using Observable?
The following should be enough.

```
include-in-header:
  - text: |
      <style>
      #quarto-announcement {
        display: none !important;
      }
      </style>
```

**Reply by cscheid** [reply]
A raw HTML block in the document would also work, something like

````
```{=html}
<style>
      #quarto-announcement {
        display: none !important;
      }
</style>
```
````

**Reply by mrworthington** [reply]
😂 sorry. My whole site is written with abunch of ojs chunks, so was mostly writing that way out of habit. The above alternatives would also work.
